composer/core/evaluator.py

```diff
@@ -118,13 +120,22 @@ def __init__(
         label: str,
         dataloader: Union[DataSpec, Iterable, Dict[str, Any]],
         metric_names: Optional[List[str]] = None,
+        metrics: Optional[Union[Metric, MetricCollection]] = None,
         subset_num_batches: Optional[int] = None,
         eval_interval: Optional[Union[int, str, Time, Callable[[State, Event], bool]]] = None,
     ):
         self.label = label
         self.dataloader = ensure_data_spec(dataloader)
 
-        self.metric_names = metric_names if metric_names else []
+        if metric_names or metrics:
+            assert (metric_names is None and metrics) or (metrics is None and metric_names)
+            if metric_names:
+                self.metric_names = metric_names
+            elif metrics:
+                if isinstance(metrics, Metric):
```

Comment:
Let's throw a DeprecationWarning for this one, tell folks to use metric_names instead in the future